agent/tool-scripts/base-tool

```diff
@@ -191,9 +191,13 @@ esac
 
 function usage {
 	printf -- "The following options are available:\n\n"
+	printf -- "Generic options - do *not* specify any of these when you register a tool:\n\n"
```

Comment:
s/Generic/Operational/ ?

Ditto in `agent/util-scripts/pbench-register-tool`.

Reply:
I was going to the opposite of "specific", but I like "operational". Back in a bit...